packages/node_modules/@ciscospark/plugin-phone/src/config.js

```diff
@@ -2,46 +2,47 @@
  * Copyright (c) 2015-2017 Cisco Systems, Inc. See LICENSE file.
  */
 
+/**
+ * @typedef {Object} PhoneConfig
+ * @property {number} [audioBandwidthLimit=64000]
+ * @property {number} [videoBandwidthLimit=1000000]
+ * @property {boolean} [enableExperimentalGroupCallingSupport=false] - Group
```

Comment:
Should `hangupIfLastActive` be defined here with the HangupBehavior type?

Reply:
the `@memberof` in `HangupBehavior` takes care of it and renders slightly better. documentation.js still seems to be my favorite doc generator, but i've definitely gotten more familiar with its rough patches than I'd like.